de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/DiskResourceMessages.properties

```diff
@@ -86,15 +86,16 @@ overWiteMetadata = Some metadata already exists. Do you want to overwrite them ?
 doiRequestFail =  DOI Request failed!
 doiRequestSuccess = Your request for DOI has been submitted! You will receive notification when there are updates to this request.
 fileUploadsSuccess = The file(s) {0,list} were successfully uploaded.
-inputLbl = Select folder(s) whose contents will processed into the HT path list file contents. Individual file(s) may also be selected. 
+inputLbl = Select folder(s) whose contents will processed into the path list file contents. Individual file(s) may also be selected. 
```

Comment:
This wasn't changed in your PR, but I think this should be "... whose contents will **be** processed..."?